version/2_7.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 
-echo "================= Installing Python 2.7.12 ==================="
+echo "================= Installing Python 2.7.14 ==================="
```

Comment:
Can you use an env var for the version? I'd like to make all the images do this consistently.

Reply:
ok